### ISSUE TITLE
Measure p90 latency and failure rate of HUD PR and commit pages

### ISSUE THREAD

**Issue by huydhn** [pytorch/test-infra#556]
### Pitch

After all recent Rockset and GitHub outages, I think we should start tracking some KPI for important developer facing HUD pages like [PR](https://hud.pytorch.org/pr/81705) and [commit](https://hud.pytorch.org/pytorch/pytorch/commit/d11d3dd036b4a7098ab3b4d333fcb556b97b4860) pages.  Here are some observations:

* It takes a noticeable amount of time to load these pages fully, from few seconds to half a minute.  Having the metric helps us balance between adding more features and performance, especially if we want to push for HUD adoption among developers (everyone love spages that load faster)

![Screen Shot 2022-08-18 at 14 53 22](https://user-images.githubusercontent.com/475357/185502076-d5f7c723-bd86-47d3-8b83-9e22b1905838.png)

* These page returns error sometime when its upstream dependencies fail, probably GitHub API but I'm not entirely sure.

### Solutions

TBD.  May be [vercel](https://vercel.com/torchci/torchci) already has these metrics somewhere that we can just tap in

**Comment by ZainRizvi** [comment]
We def need to get better about tracking metrics on our tools in general, at the least storing them somewhere to help with future debugging.

For prioritization: I'd suggest tracking usage of the tools (like who/how often people even click on HUD links) is the top priority, and then evaluating the quality of those experiences comes in second

**Comment by huydhn** [comment]
True, we have that other task to track tool usage mapped out at https://github.com/pytorch/test-infra/issues/530. So we will address it before this one
